portal/get_patient_info.php

```diff
@@ -21,11 +21,6 @@
 
 // regenerating the session id to avoid session fixation attacks
 session_regenerate_id(true);
-//
-
-// landing page definition -- where to go if something goes wrong
-$landingpage = "index.php?site=" . urlencode($_SESSION['site_id']);
```

Comment:
This change seems really odd since then the calls that use this $landingpage variable below will not be defined???

Reply:
I don't see anywhere that calls this method in a get request so I changed it to something more practical

Reply:
But then it will not be defined where it is used in this script in lines 28, 35, and 41. (Note the code here was not changed (you did the GET to SESSSION in another script which made sense there), just moved here)

Reply:
Ohhhhh now I see what ur talking about 😂 Yeah I'll go ahead and change that

Reply:
Just realized that this variable is actually a dummy and doesn't actually go to any part of the code!!!